lib/hanami/generators/app/application.rb.tt

```diff
@@ -47,22 +49,29 @@ module <%= config[:classified_app_name] %>
       # host 'example.org'
 
       # URI port used by the routing system to generate absolute URLs
-      # Argument: An object coercible to integer, default to 80 if the scheme is http and 443 if it's https
-      # This SHOULD be configured only in case the application listens to that non standard ports
+      # Argument: An object coercible to integer, default to 80 if the scheme
+      # is http and 443 if it's https
+      #
+      # This SHOULD be configured only in case the application listens to that
+      # non standard ports
```

Comment:
55 and 56 can combined this into one line by rewording to:
`     # This should only be configured if app listens to non-standard ports`